sonar-xml-plugin/src/test/java/org/sonar/plugins/xml/language/XmlTest.java

```diff
@@ -21,8 +21,8 @@
 
 import org.junit.Before;
 import org.junit.Test;
-import org.sonar.api.config.MapSettings;
 import org.sonar.api.config.Settings;
+import org.sonar.api.config.internal.MapSettings;
```

Comment:
Settings is deprecated since 6.5, I would do the move to update to `org.sonar.api.config.Configuration` in the Xml Language.